site/content/entry/yuzu-migration/index.md

```diff
@@ -0,0 +1,81 @@
++++
+date = "2019-10-04T23:51:00+05:30"
+title = "yuzu - The Migration"
+author = "CaptV0rt3x"
+forum = 150132
++++
+
+A very good day to you all ***yuz-ers*** out there! 
```

Comment:
Sounds better as `A very good day to all the`

`to you all yuz-ers` is awkward